phylo2owl.py

```diff
@@ -1,144 +1,165 @@
 #!/usr/bin/env python
 
-"""phylo2owl.py: Convert phylogenies into OWL ontologies expressed in RDF/XML."""
+"""
+phylo2owl.py: Convert phylogenies into OWL ontologies expressed in RDF/XML.
```

Comment:
Why not simply _Convert phylogenies into OWL ontologies_? That the OWL is serialized in RDF/XML seems more like current and likely temporary limitation that something important to its design or purpose.